include/chimera/util.h

```diff
@@ -315,6 +315,11 @@ std::pair<unsigned, unsigned> getFunctionArgumentRange(
  */
 bool hasNonPublicParam(const clang::CXXMethodDecl *decl);
 
+/**
+ * Converts clang operator type to the operator name in Python.
+ */
+std::string getPythonOperatorName(clang::OverloadedOperatorKind kind);
```

Comment:
Let's just call this `getOperatorName()` under the assumption it will not be python-specific later.